docs/src/main/asciidoc/rest-json-guide.adoc

```diff
@@ -474,6 +478,12 @@ was served.
 Quarkus comes with a CORS filter. Read the link:undertow-reference.html#cors-filter[Undertow Reference Documentation] to learn
 how to use it.
 
+[WARNING]
+====
+If you want to use the `keycloak` Quarkus extension in your project, you should configure CORS using the `keycloak` extension
+configuration. See link:keycloak-guide#configuring-cors[the Keycloak guide] for more details.
+====
```

Comment:
This is unrelated but it has been added in the Undertow guide but should also be there to redirect users directly to the Keycloak guide.